beta-src/src/enums/MoveOrder.ts

```diff
@@ -0,0 +1,11 @@
+enum MoveOrder {
+  MOVE_ORDER_SELECTED = "moveOrderSelected",
```

Comment:
`MOVE_ORDER_SELECTED` isn't a "move" order, you should change this enum to just be `Order` or `OrderType`

Reply:
Or maybe `Arrow` / `ArrowType` is better since move failed isn't really an order or a move either.